server/src/test/java/org/candlepin/pki/SubjectKeyIdentifierWriterTest.java

```diff
@@ -0,0 +1,17 @@
+package org.candlepin.pki;
+
+import org.candlepin.pki.impl.DefaultSubjectKeyIdentifierWriter;
+import org.hsqldb.types.Charset;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class SubjectKeyIdentifierWriterTest {
+
+    @Test
+    public void testToOctetString() throws Exception{
+        DefaultSubjectKeyIdentifierWriter writer = new DefaultSubjectKeyIdentifierWriter();
+
+        byte[] foobar_in_asn1 = {0x4, 0x6, 0x66, 0x6f, 0x6f, 0x62, 0x61, 0x72};
```

Comment:
I'd pad these to consistently use 2 hex characters and I'd leave a comment here like

"0x04 is the DER tag for an octet string and 0x06 is the subsequent length of the octet string"